src/azul/terraform.py

```diff
@@ -898,6 +898,19 @@ def tf_config(self, app_name):
         )
         locals[app_name] = json.dumps(openapi_spec)
 
+        # Chalice-generated Lambda event source mappings do not establish a TF
+        # dependency with the SQS queues they reference. Chalice composes a
+        # hard-coded ARN for the event source, so the mapping resource lacks a
+        # direct TF dependency. We replace the hard-coded ARN with a resource
+        # attribute, making the mapping resource depend on the SQS queue.
```

Comment:
```suggestion
        # Replace the hard-coded ARN emitted by Chalice with a resource reference so that the event source (the queue) is created before the event source mapping using it.
```